Dockerfile.dapper

```diff
@@ -19,6 +19,9 @@ RUN zypper -n install gcc binutils glibc-devel-static ca-certificates git-core w
 RUN zypper install -y -f docker && rpm -e --nodeps --noscripts containerd
 
 RUN curl -sLf https://github.com/rancher/machine/releases/download/${CATTLE_MACHINE_VERSION}/rancher-machine-${ARCH}.tar.gz | tar xvzf - -C /usr/bin
+RUN curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
+
+RUN zypper -n in go1.19
```

Comment:
The bci image should already be 1.19, why reinstall it?

Reply:
This was an accident I'll remove it

Reply:
Yeah for some reason this line is the v2.6 branch, and I just cherrypicked it. Weird